libs/pmp-web/auth/shell/src/lib/shell-routing.module.ts

```diff
@@ -0,0 +1,23 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { LoginComponent } from './containers/login/login.component';
+
+const routes: Routes = [
+  {
+    path: 'auth',
+    component: LoginComponent,
```

Comment:
I think it is a wrong name for an entry point component. Rename it to AuthComponent